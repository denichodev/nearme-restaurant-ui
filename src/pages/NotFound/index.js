import React from 'react';

import s from './styles.css';

const NotFound = () => {
  return (
    <div className={s.notFound}>
      <h1><i className="far fa-meh" /></h1>
      <h3>Page not found</h3>
      <p>Don't worry, we're working on this.</p>
    </div>
  );
}

export default NotFound;
