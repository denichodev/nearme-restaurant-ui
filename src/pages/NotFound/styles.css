.notFound {
  text-align: center;
  display: flex;
  align-items: center;
  justify-content: center;
  flex-flow: column;
  height: 80vh;
}

.notFound h1 {
  font-size: 64px;
  margin: 0 0 15px;
}

.notFound h3, p {
  margin: 5px 0 0;
}

.notFound p {
  font-size: 13px;
}