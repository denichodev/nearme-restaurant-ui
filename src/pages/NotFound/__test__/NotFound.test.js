import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import NotFound from '../index';

describe('<NotFound />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<NotFound />);

    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
