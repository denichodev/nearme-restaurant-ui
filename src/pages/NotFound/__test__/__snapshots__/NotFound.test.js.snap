// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> renders correctly 1`] = `
<div
  className="notFound"
>
  <h1>
    <i
      className="far fa-meh"
    />
  </h1>
  <h3>
    Page not found
  </h3>
  <p>
    Don't worry, we're working on this.
  </p>
</div>
`;
